project_generator/tool.py

```diff
@@ -113,13 +113,12 @@ def get_value(self, tool, key):
     def get_supported(self):
         return self.TOOLS.keys()
 
-def export(exporter, data, tool, env_settings):
+def export(exporter, project_workspace, tool, env_settings):
```

Comment:
Ah, I see that this exports an entire workspace. Should workspaces have env_settings and a tool associated with them as class variable?